stratum/CMakeLists.txt

```diff
@@ -433,6 +442,10 @@ elseif(DPDK_TARGET)
         target_sys
     )
 
+    if(WITH_KRNLMON)
```

Comment:
So, currently we are starting krnlmon_init as part of stratum dpdk_hal, so technically it's integrated for stratum dpdk.
Since stratum thread is blocking thread, we need to figure out how to start these 2 separate threads(stratum, krnlmon).
I will work on this after this drop, but for now it's integrated for stratum dpdk.